doc/source/user_guide/indexing.rst

```diff
@@ -1480,17 +1480,24 @@ default value.
    s.get('a')  # equivalent to s['a']
    s.get('x', default=-1)
 
-The :meth:`~pandas.DataFrame.lookup` method
--------------------------------------------
+.. _indexing.lookup
+
+Looking up values by index/column labels
```

Comment:
I think its worth saying that the .lookup method is deprecated in 1.2 (e.g. say you could formerly do this with the .lookup method)

Reply:
Added this at the bottom after the example.